<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessInfoForm</class>
 <widget class="QWidget" name="ProcessInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labProcessCountKey">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>进程数: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labProcessCount">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoAllProcess">
       <property name="text">
        <string>所有进程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rdoMyProcess">
       <property name="text">
        <string>我的进程</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tbvProcesses"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
